src/pyodide-py/pyodide/_core.py

```diff
@@ -0,0 +1,26 @@
+# type: ignore
+import platform
+
+if platform.system() == "Emscripten":
+    from _core import JsProxy, JsBoundMethod, JsException
+else:
+    # Can add shims here if we are so inclined.
+    class JsException(Exception):
```

Comment:
If it only contains exception maybe put it in `pyodide/_exceptions.py` ?

Reply:
`JsProxy` and `JsBoundMethod` aren't exceptions? Also I'm planning to automatically add `console` into this as well. `console.error` produces incredibly more helpful messages than `print`, even when just applied to python objects.